ui/apps/platform/src/Components/DateTimeField/DateTimeField.tsx

```diff
@@ -1,27 +1,17 @@
 import React, { ReactElement } from 'react';
-import { isValid, parse, format } from 'date-fns';
+import { isValid, parse } from 'date-fns';
+import { getDateTime } from 'utils/dateUtils';
 
 type DateTimeFieldProps = {
     date?: string; // ISO 8601 formatted date
-    asString?: boolean;
 };
 
-function DateTimeField({ date = '', asString = false }: DateTimeFieldProps): ReactElement {
```

Comment:
`asString` is used in pdf formatting, which was removed from VM, so this can be simplified.

I'll opt to not clean up instances of `pdf` throughout VM in the hopes that we can drop that tech debt when the deprecated sections go away.